### PR TITLE
Updating openapi-generator for PHP to 7.8.0

### PR DESCRIPTION
**Pull request by jtreminio-dropbox**

### CODE REVIEW

File: sdks/php/templates/api.mustache [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * {{classname}}
- * PHP version {{phpVersion}}
+ * PHP version 7.4
```
Comment (erkkinen) [inline sdks/php/templates/api.mustache:4]: Should we be hardcoding this? Seems useful to keep the template variable

File: sdks/php/templates/api.mustache [hunk 2]
```diff
@@ -240,76 +323,78 @@ use {{invokerPackage}}\ObjectSerializer;
...
                 $content = $response->getBody(); //stream goes to serializer
             } else {
                 $content = (string) $response->getBody();
+                if ($returnType !== 'string') {
+                    try {
+                        $content = json_decode($content, false, 512, JSON_THROW_ON_ERROR);
+                    } catch (\JsonException $exception) {
+                        throw new ApiException(
+                            sprintf(
+                                'Error JSON decoding server response (%s)',
```
Comment (erkkinen) [inline sdks/php/templates/api.mustache:389]: Might be nice to document this in the read that this error occurs

File: sdks/php/templates/api.mustache [hunk 3]
```diff
@@ -472,6 +567,9 @@ use {{invokerPackage}}\ObjectSerializer;
                         $content = $response->getBody(); //stream goes to serializer
                     } else {
                         $content = (string) $response->getBody();
+                        if ($returnType !== 'string') {
+                            $content = json_decode($content);
```
Comment (erkkinen) [inline sdks/php/templates/api.mustache:571]: Likely an upstream concern, but should we not throw on json error?

File: sdks/php/templates/api.mustache (hunk 1 above)
Comment (jtreminio-dropbox) [reply sdks/php/templates/api.mustache:4]: Comes from upstream, I'm fine keeping it as-is

**Pull request merged**